README.md

```diff
@@ -37,25 +39,25 @@ and starts exploring
 spack info <package>
 spack spec <spec>
 ```
-and building
+and installing
 ```bash
 spack install <spec>
 spack dev-build <spec>
 ```
-a package.
+packages.
 
 Updating spack-c2sm is in the hands of the user.
 ```bash
 git pull
 git submodule update --recursive
 ```
-After an update we advice to clean
+Before an update we advice to clean your instance
 ```bash
 spack uninstall -a
 spack clean -a
 rm -rf ~/.spack
 ```
-and rebuild.
+After an update we advice to rebuild packages in a new shell.
```

Comment:
Users are prone to skip the 'new shell' if we don't say why, so maybe write
```suggestion
After an update we advice to rebuild packages, preferably in a new shell so that no outdated shell variables are retained.
```